.github/workflows/test-env-deploy.yml

```diff
@@ -75,6 +75,7 @@ jobs:
         # Set test instance domain based on branch name slug
         run: |
           echo "::set-output name=domain::${{ env.GITHUB_HEAD_REF_SLUG_URL }}.dashboard.saleor.rocks"
+          echo "::set-output name=base_URL::https://${{ env.GITHUB_HEAD_REF_SLUG_URL }}.dashboard.saleor.rocks"
```

Comment:
In order to use this output in `cypress-run-critical` job you need to add `base_URL` to job outputs:
https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs